script/client/conf/file.conf

```diff
@@ -63,8 +63,4 @@ client {
   log {
     exceptionRate = 100
   }
-  support {
```

Comment:
if it not use sb starter, Isn't this switch needed?

Reply:
I will remove it.